content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -68,9 +68,13 @@ Follow these instructions to prepare a kind cluster for Istio installation.
     Deleting cluster "istio-testing" ...
     {{< /text >}}
 
+## Setup MetalLB for kind
+
+Kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types, to ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

Comment:
kind does not get capitalized at the beginning of sentences as noted on their doc pages.
```suggestion
kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types, to ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
```

Reply:
Missed the notif, will make changes